pkg/knuu/instance_helper.go

```diff
@@ -185,19 +185,16 @@ func (i *Instance) destroyPod() error {
 
 // deployService deploys the service for the instance
 func (i *Instance) deployOrPatchService() error {
-	if len(i.portsTCP) != 0 || len(i.portsUDP) != 0 {
-		logrus.Debugf("Ports not empty, deploying service for instance '%s'", i.k8sName)
-		svc, _ := k8s.GetService(k8s.Namespace(), i.k8sName)
-		if svc == nil {
-			err := i.deployService()
-			if err != nil {
-				return fmt.Errorf("error deploying service for instance '%s': %w", i.k8sName, err)
-			}
-		} else if svc != nil {
-			err := i.patchService()
-			if err != nil {
-				return fmt.Errorf("error patching service for instance '%s': %w", i.k8sName, err)
-			}
+	svc, _ := k8s.GetService(k8s.Namespace(), i.k8sName)
+	if svc == nil {
+		err := i.deployService()
+		if err != nil {
+			return fmt.Errorf("error deploying service for instance '%s': %w", i.k8sName, err)
+		}
+	} else if svc != nil {
```

Comment:
just for curiosity
can `svc` be something different than `nil`? 
I wonder why the `else if svc != nil`

Reply:
please consider looking to switch statements as I understood that there is a 3rd option for svc variable.

Reply:
There is not 3rd option, good catch!
https://github.com/celestiaorg/knuu/pull/174/commits/4d826dc2b0b9f7f08eafc85a6ff7ebf561821dde